examples/globe_vector.html

```diff
@@ -75,53 +77,79 @@
             itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
             itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
 
+            var promises = [];
+
+            // Fetch, Parse and Convert by iTowns
             var kmlSource = new itowns.FileSource({
-                    url: 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/croquis.kml',
-                    projection: 'EPSG:4326',
-                }, view.tileLayer.extent.crs());
+                url: 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/croquis.kml',
+                projection: 'EPSG:4326',
+                fetcher: itowns.Fetcher.xml,
+                parser: itowns.KMLParser.parse,
+            });
 
             var kmlLayer = new itowns.ColorLayer('Kml', {
                 name: 'kml',
                 transparent: true,
+                projection: view.tileLayer.extent.crs(),
```

Comment:
it's not up to the user to specify the target projection, this should be done by itowns, when adding the layer.

Reply:
Removed